.github/ISSUE_TEMPLATE/xtable_bug_report.yml

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+name: XTable bug report 🐞
+description: Problems, bugs and issues with Apache XTable
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for submitting feedback to XTable to help XTable develop better.
+
+        If it is an idea or help wanted, please go to:
+
+        1. [Dev Mail List](mailto:dev@xtable.apache.org): This will be your FASTEST way to get help！[How to subscribe](mailto:dev-subscribe@xtable.apache.org)
+        2. [Github Discussion](https://github.com/apache/incubator-xtable/discussions)
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please make sure to search in the [issues](https://github.com/apache/incubator-xtable/issues?q=is%3Aissue) first to see
+        whether the same issue was reported already.
+      options:
+        - label: >
+            I had searched in the [issues](https://github.com/apache/incubator-xtable/issues?q=is%3Aissue) and found no similar
+            issues.
+          required: true
+  - type: textarea
+    attributes:
+      label: Please describe the bug 🐞
+      description: >
+        Please describe the problem, what to expect, and how to reproduce.
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Are you willing to submit PR?
+      description: >
+        We very much look forward to developers or users to help solve the XTable problem together.
+        If you are willing to submit a PR to fix this problem, please tick it.
+      options:
+        - label: Yes I am willing to submit a PR!
```

Comment:
Should we add an option like "I am willing to submit a PR but need help getting started"?

Delta Lake has an option like this and it can be useful for finding new people trying to ramp up on the project

Reply:
Agree, good idea.